s4e-backend/src/main/java/pl/cyfronet/s4e/listener/InvitationListener.java

```diff
@@ -64,11 +69,8 @@ private void send(Invitation invitation, Locale locale) {
 
         ctx.setVariable("email", invitation.getEmail());
         ctx.setVariable("institutionName", invitation.getInstitution().getName());
-
-        String confirmUrl = mailHelper.prefixWithDomain("/login?token=" + invitation.getToken());
-        String rejectUrl = mailHelper.prefixWithDomain("/login?reject=''&token=" + invitation.getToken());
-        ctx.setVariable("confirmUrl", confirmUrl);
-        ctx.setVariable("rejectUrl", rejectUrl);
+        ctx.setVariable("token", invitation.getToken());
+        ctx.setVariable("urlDomain", this.mailProperties.getUrlDomain());
```

Comment:
I don't think you need this, as `urlDomain` is injected in `MailHelper::injectCommonVariables`